src/Events/Editor/Full_Site/Templates/single-event.html

```diff
@@ -0,0 +1,3 @@
+<!-- wp:template-part {"slug":"header","tagName":"header"} /-->
+<!-- wp:tribe/single-event {} /-->
```

Comment:
```
<!-- wp:tribe/single-event {} /-->
```
More of a question here.

Is the `tribe/single-event` a reference to our other block? I assume we will not be using the existing blocks for this. 

Also wondering if we should name it like `tec-events/...` or what our naming convention might be (vs `tribe`)?

Reply:
`tribe/` is the old namespace for blocks, we will need at some point to create a new one and have the old ones map to this new one.